code/api/src/modules/user/mutations.js

```diff
@@ -22,11 +22,48 @@ export const userSignup = {
     password: {
       name: 'password',
       type: GraphQLString
+    },
+
+    stylePreference: {
+      name: 'stylePreference',
+      type: GraphQLString
     }
   },
   resolve: create
 }
 
+// Update
+export const userUpdate = {
+  type: UserType,
+  args: {
+    id: {
+      name: 'id',
+      type: GraphQLInt
+    },
+
+    name: {
+      name: 'name',
+      type: GraphQLString
+    },
+
+    email: {
+      name: 'email',
+      type: GraphQLString
+    },
+
+    password: {
+      name: 'password',
+      type: GraphQLString
+    },
+
+    stylePreference: {
+      name: 'stylePreference',
+      type: GraphQLString
```

Comment:
This might need to be changed to array if thats how we choose to handle the logic